src/manager.hpp

```diff
@@ -54,6 +58,35 @@ class Manager
         return std::ranges::contains(_dataSyncConfiguration, dataSyncCfg);
     }
 
+    /**
+     * @brief Initiates a full synchronization between two BMCs.
+     *
+     *        - This method is responsible for initiating the  Full
+     *          synchronization process between two BMCs.
+     *        - The sync process is handled asynchronously.
+     *
+     */
+    sdbusplus::async::task<> startFullSync();
+
+    /**
+     * @brief Helper API that retrieves the sibling BMC IP and returns its
+     *        availability as a boolean.
+     *
+     * @return True if sibling BMC IP is empty; otherwise False.
+     */
+    bool isSiblingBmcAvailable()
```

Comment:
i think I'm confused on how an empty sibling IP means the sibling BMC is available, instead of the opposite.